src/tools/cephfs/cephfs-shell

```diff
@@ -112,7 +112,7 @@ def ls(path, opts=''):
                 elif almost_all and dent.d_name in (b'.', b'..'):
                     continue
                 yield dent
-    except cephfs.ObjectNotFound:
+    except libcephfs.ObjectNotFound:
         return []
```

Comment:
Let's move the warning to this block so any command using this `ls` method will see:

`self.perror('{}: no such directory exists'.format(path), end='\n', apply_style=True)`